helm/alfresco-content-services/templates/config-repository.yaml

```diff
@@ -40,13 +40,19 @@ data:
       {{- if index .Values "alfresco-search" "enabled" }}
       -Dsolr.host={{ template "alfresco-search.host" . }}
       -Dsolr.port={{ template "alfresco-search.port" . }}
+      -Dsolr.secureComms={{ .Values.global.tracking.auth | default "none" }}
```

Comment:
It was explicitly requested by architects (@skopf) that the default is set to `secret` in case search2 is enabled.

Reply:
that is true now, because before was set regardless of search2 enabled or not (or maybe those ifs are very hard to read for me?)

Reply:
yes, the if logic works best like this but the default is still `none`. But tbh that was also the case before... up to you.